packages/http/README.md

```diff
@@ -1,152 +1,148 @@
 # Prism HTTP
 
-This package provides HTTP APIs for forwarding and mocking requests.
-
-## HTTP Forwarder
-
-> Note: The current API is still experimental - changes can be introduced in future.
-
-Module used to forward provided request. Is exposes single `forward` method and can be initialized with `IHttpOperation` and `IHttpRequest`.
+This package provides a http client featuring the ability to:
+
+- mock requests instead of hitting the server
+- validate the request and the response according to an openapi spec
+
+# Basic Usages
+
+> Note: examples in this document use the following spec
+
+```yaml
+---
+openapi: 3.0.2
+paths:
+  /todos:
+    get:
+      parameters:
+        - name: title
+          in: query
+          style: form
+          schema:
+            enum:
+              - eat
+              - drink
+      responses:
+        200:
+          description: Get Todo Items
+```
 
-### Forwarder props
+## I want to use Prism to mock responses
 
-Object containing:
+[Try it!](https://repl.it/@ChrisMiaskowski/prism-http-client-basic-mocking);
 
-- `resource` containing `IHttpOperation` _(optional)_
-- `input` `IHttpRequest` wrapped with `IPrismInput`
-- `timeout` overrides default timeout _(optional)_
-- `cancelToken` cancellation token _(optional)_
+```javascript
+const Prism = require('@stoplight/prism-http');
+const path = require('path');
 
-Properties used from `resource: IHttpOperation`:
+// Create Prism instance and configure it as a mocker generating static examples
+const config = { mock: { dynamic: false } };
+const prism = Prism.createInstance(config);
 
-- `servers[]` - if not empty the first element will be used as server url
+// Load openapi spec file
+const specPath = path.resolve(process.cwd(), 'basic.oas3.yaml');
+prism
+  .load({ path: specPath })
+  .then(() => {
+    // Make a "GET /todos" request
+    return prism.process({
+      method: 'get',
+      url: {
+        path: '/todos',
+      },
+    });
+  })
+  .then(prismResponse => {
+    console.log(prismResponse.output);
+  });
+```
 
-The type of `input` parameter `IPrismInput<IHttpRequest>` is defined as:
+Output
 
-```ts
-interface IPrismInput<IHttpRequest> {
-  data: IHttpRequest;
-  validations: {
-    input: IPrismDiagnostic[];
+```bash
+{ statusCode: 200,
+  headers: { 'Content-type': 'text/plain' },
+  body: undefined }
```

Comment:
Do we really want an undefined body as the primary example for a response mocking tool? If feels like we are missing an opportunity to actually show off what prism does.

Reply:
Yeah good point :P